docs/docs/about/modelling/setup/models/compressor_modelling/compressor_models_types/simplified_variable_speed_compressor_train_model.md

```diff
@@ -8,7 +8,7 @@ are assumed based on an assumption of equal pressure fractions for each stage. B
 calculated independently for each compressor as if it was a standalone compressor, neglecting that they are in fact on
 the same shaft and thus have a common speed.
 
-This model supports both `user defined compressor charts` and
+This model supports only 
```

Comment:
Is this a change of clarification?

Reply:
It is a change. Now only generic charts are supported. Previously SINLGE_SPEED and VARIABLE_SPEED were allowed in addition.

Reply:
Ok, so potentially breaking? Then that should be mentioned too, if it hasnt